### PR TITLE
Add includeEncrypted parameter

### PR DESCRIPTION
**Pull request by NikolasStelmakh**

### CODE REVIEW

File: src/transactions/queries/ReceivedFioRequests.ts [hunk 1]
```diff
@@ -36,7 +38,7 @@ export class ReceivedFioRequests extends Query<ReceivedFioRequestsResponse> {
           value.content = content
           requests.push(value)
         } catch (e) {
-          //
+          if (this.includeEncrypted) { requests.push(value) }
```
Comment (andreyvEze) [inline src/transactions/queries/ReceivedFioRequests.ts]: remove `{` `}`

File: src/FIOSDK.ts [hunk 2]
```diff
@@ -601,7 +600,7 @@ export class FIOSDK {
       [{
         chain_code: chainCode,
         token_code: tokenCode,
-        public_address: publicAddress
+        public_address: publicAddress,
```
Comment (andreyvEze) [inline src/FIOSDK.ts]: please remove all of the changes that not relate to the feature, and create separate PR for it

File: src/transactions/queries/SentFioRequests.ts [hunk 3]
```diff
@@ -36,7 +38,7 @@ export class SentFioRequests extends Query<SentFioRequestResponse> {
           value.content = content
           requests.push(value)
         } catch (e) {
-          //
+          if (this.includeEncrypted) { requests.push(value) }
```
Comment (andreyvEze) [inline src/transactions/queries/SentFioRequests.ts]: remove `{` `}`

File: src/transactions/queries/ReceivedFioRequests.ts (hunk 1 above)
Comment (NikolasStelmakh) [reply src/transactions/queries/ReceivedFioRequests.ts]: ok, but { } is required in the tslint rules

**Review by andreyvEze — APPROVED** [review]

**Review by ericbutz — APPROVED** [review]

**Pull request merged**